distr_tracing/distr_tracing_rn/distr-tracing-rn-2-9.adoc

```diff
@@ -0,0 +1,196 @@
+:_content-type: ASSEMBLY
+[id="distributed-tracing-rn-2-9"]
+= Release notes for {DTProductName} 2.9
+include::_attributes/common-attributes.adoc[]
+:context: distributed-tracing-rn-2-9
+
+toc::[]
+
+include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
+
+== Component versions in the {DTProductName} 2.9
+
+[options="header"]
+|===
+|Operator |Component |Version
+|{JaegerName}
+|Jaeger
+|1.47.0
+
+|{OTELName}
+|OpenTelemetry
+|0.81.0
+
+|{TempoName}
+|Tempo
+|2.1.1
+|===
+
+== {JaegerName}
+
+=== New features and enhancements
+
+* None.
+
+//=== Technology Preview features
+//not for 2.9
+
+=== Bug fixes
+
+* Before this update, connection was refused due to a missing gRPC port on the `jaeger-query` deployment. This issue was causing the following error: *transport: Error while dialing: dial tcp :16685: connect: connection refused*. With this update, the Jaeger Query gRPC port (16685) is successfully exposed on the Jaeger Query service. (link:https://issues.redhat.com/browse/TRACING-3322[TRACING-3322])
```

Comment:
[System messages](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#system-messages)
```suggestion
* Before this update, connection was refused due to a missing gRPC port on the `jaeger-query` deployment. This issue resulted in `transport: Error while dialing: dial tcp :16685: connect: connection refused` error message. With this update, the Jaeger Query gRPC port (16685) is successfully exposed on the Jaeger Query service. (link:https://issues.redhat.com/browse/TRACING-3322[TRACING-3322])
```

Reply:
Thank you. Applied manually.